### PR TITLE
Improve layout of ctf_map GUI

### PR DESCRIPTION
**Pull request by ClobberXD**
- Improve layout consistency.
- Add a text field for `initial_stuff`.

screenshot_20181111_163625

### CODE REVIEW

**Comment by AKryukov92** [comment]
Do we have some kind of tutorial about this formspec?
Which delimiter should be used if I want to define multiple starting items?

**Comment by ClobberXD** [comment]
> Do we have some kind of tutorial about this formspec?

https://youtu.be/orBsC9wViUw

> Which delimiter should be used if I want to define multiple starting items?

Multiple items are comma-separated. I initially did want to add this clarification, but there's just not enough space. Should I increase the formspec height and re-add it?

**Comment by rubenwardy** [comment]
Maybe also add horizontal and vertical rules to split up sections? Can be done with a stretched `background[]`

File: mods/ctf_map/map_maker.lua [hunk 1]
```diff
@@ -149,30 +151,33 @@ local function show_gui(name)
...
+		"label[4.8,3.5;", minetest.formspec_escape(get_flag_status()), "]",
+		"button[4.8,4.2;3.5,1;giveme;Giveme Flags]",
 
-		"label[0,5.5;4. Meta Data]",
-		"field[0.4,6.5;7.5,1;title;Title;" , minetest.formspec_escape(maptitle), "]",
-		"field[0.4,7.8;3.75,1;name;File Name;" , minetest.formspec_escape(mapname), "]",
-		"field[4.15,7.8;3.75,1;author;Author;", minetest.formspec_escape(mapauthor), "]",
+		"label[0,5.2;4. Meta Data]",
+		"field[0.4,6.2;8.5,1;title;Title;" , minetest.formspec_escape(maptitle), "]",
+		"field[0.4,7.3;8.5,1;initial_stuff;Initial Stuff (optional): ",
```
Comment (rubenwardy) [inline mods/ctf_map/map_maker.lua]:
```
Stuff to give on (re)spawn, comma separated
```

**Comment by ClobberXD** [comment]
I'd love to add some separators, thanks for the suggestion. :)

**Comment by ClobberXD** [comment]
- Improved the label of `initial_stuff` field.
- Added separators using `box[]` element, with the colorstring `#111111BB`.

Updated screenshot in first post.

File: mods/ctf_map/map_maker.lua (hunk 1 above)
Comment (ClobberXD) [reply mods/ctf_map/map_maker.lua]: Added. Please review. :)

**Pull request merged**